app/assets/stylesheets/_header.sass

```diff
@@ -0,0 +1,5 @@
+#unitern-navbar
```

Comment:
The file name with the _ is usually only used for partials. This CSS could be pretty much a standalone too. I usually only use _ is CSS if it is just variables and can't be used on its own. Just a preference.